mne_bids/tests/test_utils.py

```diff
@@ -44,7 +44,7 @@ def test_get_ch_type_mapping():
 
 
 def test_handle_kind():
-    """Test the automatic extraction of kind from the data."""
+    """Test the automatic extraction of str_suffix from the data."""
```

Comment:
```suggestion
    """Test the automatic extraction of suffix from the data."""
```

function name needs also to change